README.md

```diff
@@ -15,6 +17,8 @@ checkNpmVersions({
 const Griddle = require('griddle-react');
 ```
 
+>This must be run on the server so that it has access to the package.json of the package where it can examine the version that is installed.
+
```

Comment:
Add section for client-only package instructions, since you will need to statically import either the whole package, or just the package.json for the module to be included in the server bundle so we can check that it exists.
```
checkNpmVersions({
  'client-only': '1.3.x'
}, 'my:package');
require('client-only/package.json');
```

Reply:
What about adding a check to make sure we are in development and effectively turn the `checkNpmVersions` into a noop for production?